docs/guides/hooks-list.rst

```diff
@@ -478,6 +478,12 @@ Embed
 
 **embed_get_upload_sections, all**
 
+Groups
+------
+
+**profile_buttons, group**
+	Filters buttons to be registered in the title menu of the group profile page
```

Comment:
Either work. I'd much rather standardize on one type, which would make it easier for hook handlers to examine what's there. Let's throw if anything other than arrays are given?